spec/dummy/app/views/layouts/application.html.erb

```diff
@@ -11,4 +11,6 @@
       <%= javascript_include_tag 'requirejs/require', data: {main: javascript_path('application')} %>
     <!--<![endif]-->
   </body>
+
+  <%= render template: parent_template %>
```

Comment:
@tomas-stefano do you have any idea as to why we render the _parent_template_ in the spec dummy? I think we can close this PR if this is not really required. I don't understand why it's included and I'll love an explanation :)

Reply:
Sure, np. The code that includes the parent template on the dummy represents:

1) *The application that will mount wpcc* should implement the parent template method (e.g the frontend/site should implement parent_template method)
2) The parent template method is the main layout that the application that will mount wpcc will use after rendering the engine layout (e.g the wpcc layout will be rendered and after will render the parent template which is the site layout - this explain why the wpcc layout has only content for definitions and the site layout the yield for those content for).

In the case of our site, the parent template is implemented [here](https://github.com/moneyadviceservice/frontend/blob/master/app/controllers/application_controller.rb#L32).

Yes, I think we should close this PR.

Reply:
Thanks for the clear explanation @tomas-stefano